### PR TITLE
pqdict pip package

### PR DESCRIPTION
**Pull request by fcolas**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2811,6 +2811,10 @@ python-plyfile-pip:
 python-poster:
   debian: [python-poster]
   ubuntu: [python-poster]
+python-pqdict:
```
Comment (clalancette) [inline rosdep/python.yaml]: For a pip package, we require a `-pip` suffix for the key.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2811,6 +2811,10 @@ python-plyfile-pip:
...
+  ubuntu:
```
Comment (clalancette) [inline rosdep/python.yaml]: Also, if we are going to make it a `-pip` key, we may as well enable it for fedora, debian, and osx as well.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (fcolas) [reply rosdep/python.yaml]: Done.

File: rosdep/python.yaml (hunk 2 above)
Comment (fcolas) [reply rosdep/python.yaml]: Done. Thanks for the review.

**Review by clalancette — APPROVED** [review]

**Pull request merged**